backend/benefit/applications/api/v1/views.py

```diff
@@ -491,3 +496,76 @@ def _csv_pdf_response(
         )
         response["Content-Disposition"] = f"attachment; filename={zip_filename}"
         return response
+
+
+class PrintDetail(APIView):
+    renderer_classes = [TemplateHTMLRenderer]
+    permission_classes = [BFIsApplicant]
+
+    def not_found(self):
```

Comment:
Hmm, how is this view actually called? Is the user redirected to the URL, or is this called from the frontend? I think that there might be a possibility that sending HTML response might mess up frontends if this is used from there.

Reply:
No HTML response is sent, only PDF as bytes and MIME-type set to PDF.

Reply:
Added `settings.DEBUG` condition to allow `HTML` rendering.

Reply:
I actually meant this 404 view.  But maybe or frontends can handle it.